include/constants/dungeon.h

```diff
@@ -108,4 +108,12 @@ enum DungeonID
     NUM_DUNGEONS
 };
 
+static inline bool8 IsClientOrTeamBase(s32 joinedDungeon)
```

Comment:
This function occurs in a number of other places under the name `JoinLocationCannotUseItems`. Perhaps those usages can be combined here too?

Reply:
Sure, I could do that. What about the name, which one you find better? Or maybe you have a better idea for the name : D

Reply:
I went with IsClientOrTeamBase

Reply:
I originally named `JoinLocationCannotUseItems` for an inline function related to items, but we've now found other occurrences that aren't item-related, like determining whether a Pokémon can gain experience. I'm not sure where `DUNGEON_RESCUE_TEAM_BASE` is used, so I like a literal name like `IsClientOrTeamBase` better as well.